ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/JdbcLedgerDao.scala

```diff
@@ -654,6 +657,17 @@ private class JdbcLedgerDao(
       eventProcessingParallelism = eventsProcessingParallelism,
       metrics = metrics,
       lfValueTranslation = translation,
+      acsReader = new FilterTableACSReader(
+        dispatcher = dbDispatcher,
+        queryNonPruned = queryNonPruned,
+        eventStorageBackend = readStorageBackend.eventStorageBackend,
+        pageSize = eventsPageSize,
+        idPageSize = eventsPageSize * 20,
+        idFetchingParallelism = 2,
+        acsFetchingparallelism = 2,
```

Comment:
idPageSize: id retrival is so fast, returns so little data, with 20 it is much faster than the batch retrieval (second phase) - so empirical.
acsFetchingparallelism: scales very well, makes able to amortize call times and sequential processing overhead quite nicely already, it is fairly small not to endanger the server
idFetchingParallelism: this could be probably one in most cases, but as the stream starts, there is a spike in id fetching, keeping this also higher enables a lower lag